schedule/yam/agama-from-usb.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Please use `_` instead of `-` (I know :) it get confusing a bit while we develop in TS).